docs/design/persistent-storage.md

```diff
@@ -0,0 +1,83 @@
+# PersistentVolume
+
+This document proposes a model for managing persistent, cluster-scoped storage for applications requiring long lived data.
+
+### tl;dr
+
+Two new kinds:
+
+A `PersistentVolume` is created by a cluster admin and is a piece of persistent storage exposed as a volume.  It is resource analogous to a node.
+
+A `PersistentVolumeClaim` is a user's request for a persistent volume to use in a pod. It is a request for a resource analogous to a pod.  
+
+One new component:
+
+`PersistentVolumeController` watches for new volumes to put into the pool.  Also watches for claims by users and binds them with available volumes in the pool.
```

Comment:
Renamed to `PersistentVolumeManager`, because it manages volumes in the system (unless there's a name y'all prefer).  Also removed references to a "pool" from the docs, because we're not introducing a new thing.  Volumes created by an admin exist and are managed.  There is no "pool".